src/tests/unit/tests/assessments/custom-widgets/custom-widgets-column-renderer.test.tsx

```diff
@@ -59,7 +59,7 @@ describe('CustomWidgetsColumnRenderer', () => {
         const expectedValues = [
             { designPattern: 'Accordion', URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#accordion' },
             { designPattern: 'Button', URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#button' },
-            { designPattern: 'Dislosure (Show/Hide)', URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure' },
+            { designPattern: 'Disclosure (Show/Hide)', URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure' },
```

Comment:
This seems to be a good candidate to be converted to snapshot test.

Reply:
Agreed, but leaving for separate PR